svelte-cloudinary/src/lib/components/CldUploadWidget.svelte

```diff
@@ -145,14 +156,98 @@
 		}
 	}
 	onMount(() => {
-		if(!window.cloudinary?.createUploadWidget) {
-			return loadCloudinary({ onLoad: handleOnLoad, onError: handleError })
+		if (!window.cloudinary?.createUploadWidget) {
+			return loadCloudinary({ onLoad: handleOnLoad, onError: handleError });
 		}
 		return handleOnLoad();
 	});
 
-</script>
+	/**
+	 * Instance Methods
+	 * Gives the ability to interface directly with the Upload Widget instance methods like open and close
+	 * https://cloudinary.com/documentation/upload_widget_reference#instance_methods
+	 */
+
+	function invokeInstanceMethod<TMethod extends keyof CldUploadWidgetInstanceMethods>(
+		method: TMethod,
+		options: Parameters<CldUploadWidgetInstanceMethods[TMethod]> = [] as Parameters<
+			CldUploadWidgetInstanceMethods[TMethod]
+		>
+	) {
+		if (!widget) {
+			widget = createWidget();
+		}
+
+		if (typeof widget?.[method] === 'function') {
+			const validOptions = options.filter((option) => typeof option !== 'undefined').filter(option => !(option instanceof PointerEvent));
```

Comment:
do you think the specificity of filtering on `PointerEvent` will be enough safeguard for the future? wondering if it's going to just wind up back here again 🤔  it may be worth checking what's being sent through thats not valid and make sure it doesnt get here in the first place

here's the latest in Next Cloudinary that handles this: https://github.com/cloudinary-community/next-cloudinary/blob/main/next-cloudinary/src/components/CldUploadWidget/CldUploadWidget.tsx#L166

the options need to be serializable so something must be getting through somehow thats not

Reply:
maybe we can add something like

```js
const validOptions = JSON.parse(JSON.stringify(options))
```

That will remove all non-serializable data